lib/mpl_toolkits/axes_grid1/axes_rgb.py

```diff
@@ -112,47 +117,12 @@ def __init__(self, *args, pad=0, add_all=True, **kwargs):
         **kwargs
             Unpacked into axes_class() init for RGB, R, G, B axes
         """
-        try:
-            axes_class = kwargs.pop("axes_class", self._defaultAxesClass)
-        except AttributeError as err:
-            raise AttributeError(
-                'A subclass of RGBAxesBase must have a _defaultAxesClass '
-                'attribute. If you are not sure which axes class to use, '
-                'consider using mpl_toolkits.axes_grid1.mpl_axes.Axes.'
-            ) from err
-
-        ax = axes_class(*args, **kwargs)
-
-        divider = make_axes_locatable(ax)
-
-        pad_size = pad * Size.AxesY(ax)
-
-        xsize = ((1-2*pad)/3) * Size.AxesX(ax)
-        ysize = ((1-2*pad)/3) * Size.AxesY(ax)
-
-        divider.set_horizontal([Size.AxesX(ax), pad_size, xsize])
-        divider.set_vertical([ysize, pad_size, ysize, pad_size, ysize])
-
-        ax.set_axes_locator(divider.new_locator(0, 0, ny1=-1))
-
-        ax_rgb = []
-        for ny in [4, 2, 0]:
-            ax1 = axes_class(ax.get_figure(),
-                             ax.get_position(original=True),
-                             sharex=ax, sharey=ax, **kwargs)
```

Comment:
This `kwargs` is missing from `make_rgb_axes`.

Reply:
good catch, fixed.